### DISCUSSION TITLE
Acceptance test

### DISCUSSION THREAD

**Discussion by 13122759676** [kgrzybek/modular-monolith-with-ddd#111] [General]
Thank you very much for your project, it has changed my career!
I learned from the book that acceptance testing is important. Is that true？ I want to learn more from you：)

**Comment by kgrzybek** [comment]
Hi,

What do you mean by "acceptance tests"? Do you mean manual User Acceptance Tests (UAT) or automated acceptance tests?

Both are important and have different motivations:
1. Automated acceptances tests give you confidence that the system works as expected, protect against regression and are executable documentation of the system. They are part of CI/CD
2. User Acceptance Tests are manual tests performed by users of the system. They can test if the system allows them to achieve their goals, has good UX and so on.
